docs/api-guide/permissions.md

```diff
@@ -43,8 +43,8 @@ This will either raise a `PermissionDenied` or `NotAuthenticated` exception, or
 
 For example:
 
-    def get_object(self):
-        obj = get_object_or_404(self.get_queryset())
+    def get_object(self, pk):
+        obj = get_object_or_404(self.get_queryset(), pk=pk)
```

Comment:
The [default `get_object()` implementation](https://github.com/encode/django-rest-framework/blob/master/rest_framework/generics.py#L97-L98) does this: 


```python 
        filter_kwargs = {self.lookup_field: self.kwargs[lookup_url_kwarg]}
        obj = get_object_or_404(queryset, **filter_kwargs)
```

That's a bit generic for the point being made here so I'd be happy with just: 

```
    obj = get_object_or_404(queryset, pk=self.kwargs["pk"])
```